CHANGELOG.txt

```diff
@@ -33,7 +33,8 @@ master:
  - obspy.scripts:
    * obspy-scan command line script now also plots and prints overlaps
      alongside gaps (see #1366)
-
+ - obspy.imaging.waveform
+   * Support for filling the signal wiglets when plotting sections (horizontal and vertical).
```

Comment:
Please make sure this line does not exceed 79 characters and also add the number of this PR `(see #1445)`.